### ISSUE TITLE
To, CC, BCC buttons disappear in Draft

### ISSUE THREAD

**Issue by StefanAbel-OTOBO** [RotherOSS/otobo#2388]
When you are in TicketZoom and click on Reply, you see that you can move the mail addresses via buttons to the To-field, to the CC-field and to the BCC-field.
When you now save your reply as a draft and then open it again, those buttons are gone (not there anymore).

Tested 10.1.7 (e.g. ov153)

**Comment by bschmalhofer** [comment]
The buttons for moving to 'To', 'CC', 'BCC' were added for #1448 . These buttons are included when the mail address was added via JavaScript. But not when the addresses are already known when the page is created. This causes the reported glitch.

The relevant commit is be234fbe451 . This patch adds 6 lines in the template-div in several templates. The same six lines have to be added again in the section "MultipleCustomer".

**Comment by bschmalhofer** [comment]
I tried to fix this, but apparenty I'm missing something. I expected that this works after adding the buttons and the event listeners. The current state is now available on the branch https://github.com/RotherOSS/otobo/tree/issue-%232388-to_cc_bcc.

@stefanhaerter , @svenoe : It's propably an easy fix, could you one of you take a look at the branch?

**Comment by stefanhaerter** [comment]
I tried it out and found the error message `TargetNS.AddTicketCustomer is not a function`. Thus, I changed

```
TargetNS.AddTicketCustomer(TargetField, MoveCustomerVal, MoveCustomerKey);
```

to

```
Core.Agent.CustomerSearch.AddTicketCustomer(TargetField, MoveCustomerVal, MoveCustomerKey);
```

and it worked as far as I tested it. I tried out every interface except AgentTicketPhone, where, to the best of my knowledge, the fields To, Cc and Bcc aren't present in general. Thus, I would propose to dismiss the changes made in `Core.Agent.TicketPhone.js`.
